cypress/e2e/modals.cy.ts

```diff
@@ -217,6 +218,52 @@ describe("modals", () => {
         metadata: { "maputnik:renderer": "ol" },
       });
     });
+
+
+
+    it("inlcude API key when change renderer", () => {
+
+      
+      
+      when.click("modal:settings.close-modal")
+      when.click("nav:open");
+
+
+      // when.click("aria-label="MapTiler Basic"
+
+      cy.get('[aria-label="MapTiler Basic"]').should('exist').click();
```

Comment:
Use helper or driver

Reply:
Just so I don't put things completely random, what's the gist of this driver / helper? I assume it's to increase code reuse, and the helper is in the driver instance? modal/maputnik driver, what's the difference there?

Reply:
Check out the architecture documents in the readme of the following package:
https://www.npmjs.com/package/@shellygo/cypress-test-utils
The driver should communicate with cypress - have `cy.something` methods (in most cases, there's no need to add anything here as part of this project)
The helper should facilitate "PM" like options - clickThat, openModal, navigateToAScreen, and the test itself should be using the helper to create a test that sounds like an acceptance criteria - `given.somthing(); when.IDoSomething(); then.IshouldBeSomething();`
It might not be perfect or exactly that in all tests, but this is the concept.

Reply:
Getting a hang of it - I managed to do that now.